.gitignore

```diff
@@ -110,6 +113,6 @@ $RECYCLE.BIN/
 # !Protogame/Run/*.dll
 **/Temporary/*
 .vs
-!**/Run/*.exe
+**/Run/*.exe
```

Comment:
Just remove this line rather than just removing the not symbol

Reply:
done